Scarb.toml

```diff
@@ -11,5 +11,10 @@ license-file = "LICENSE"
 
 [workspace.dependencies]
 cairo_test = "2.8.0"
-
 shinigami_engine = { git = "https://github.com/keep-starknet-strange/shinigami.git", rev = "9e7692d" }
+
+[profile.cairo1-run.cairo]
+# https://github.com/lambdaclass/cairo-vm/issues/1745
+enable-gas = false
```

Comment:
Out of curiosity, what does this disable?

Reply:
I think gas related instructions, but scarb cairo-run doesn't work without `enable-gas = true` even if you remove `get_gas_available` function usage